LongWarOfTheChosen/Config/XComLW_UI.ini

```diff
@@ -60,3 +60,9 @@ LEVEL_ICON_OFFSET_CTRL = 0					; Controller settings
 LEVEL_ICON_SIZE_CTRL = 24
 LIST_ITEM_FONT_SIZE_CTRL = 24
 LIST_ITEM_FONT_SIZE_FANCY_CTRL = 26
+
+[LW_Overhaul.UIStrategyMapItem_Region_LW]
+HIDE_REGION_TOOLTIPS_CTRL = true			; Hides the region tooltips when using a controller. The problem with these tooltips is that they can be quite long and uninformative.
+
+[LW_Overhaul.UIStrategyMapItem_Mission_LW]
+SHOW_MISSION_TOOLTIPS_CTRL = true			; Shows tooltips for missions when using a controller. These tooltips are the same as what you would see on the bottom UI bar when using a mouse & keyboard.
```

Comment:
I would prefer these to have a consistent name, so either `SHOW_...` or `HIDE_...`. I lean towards `SHOW_..._CTRL`.

Reply:
Will do.